config/nuts/discovery/test:SharedCarePlanning2024.json

```diff
@@ -1,121 +1,109 @@
-{
-  "id": "test:SharedCarePlanning2024",
```

Comment:
same here, it's hard to see actual changes, can you try and rollback any reformatting?